release-notes/6.0/linux-packages.md

```diff
@@ -0,0 +1,99 @@
+# Linux package dependencies
+
+.NET 6 has several dependencies that must be satisfied to run .NET apps. The commands to install these libraries are listed for multiple distributions.
+
+Feel free to contribute packages for distributions not (yet) listed in this document, including ones not supported by the .NET Team.
+
+Tips:
+
+- [runtime-deps container images](https://github.com/dotnet/dotnet-docker/tree/main/src/runtime-deps) installs these same packages. You can look at those dockerfiles.
+- [pkgs.org](https://pkgs.org/) is a useful site for searching for packages, to find the one for your distribution.
+
+## Packages
+
+.NET depends on the following packages.
+
+- [GNU C Library (glibc)](https://www.gnu.org/software/libc/libc.html)
+- [GNU C++ Library](https://gcc.gnu.org/onlinedocs/libstdc++/)
+- [GCC low-level runtime library](https://gcc.gnu.org/onlinedocs/gccint/Libgcc.html)
+- [ICU](http://site.icu-project.org/)
+- [Kerberos](http://web.mit.edu/kerberos/)
+- [Open SSL](https://www.openssl.org/)
+- [zlib compression library](https://www.zlib.net/)
+
+You do not need to install ICU if you [enable globalization invariant mode](https://github.com/dotnet/runtime/blob/main/docs/design/features/globalization-invariant-mode.md#enabling-the-invariant-mode).
+
+If your app relies on `https` endpoints, you'll also need to install `ca-certificates`.
+
+## Alpine 3.13
```

Comment:
The distros listed here don't seem to match the distros listed in the [supported-os.md](https://github.com/dotnet/core/pull/6100/files#diff-30c5b4a9fd2a15bde26a66f33b74c27dd20ce62a36eeda73b2eaa317147fb26eR24).  This seems like a disconnect.

Reply:
You are right. I'll be adding those. I wasn't going to block the first merge on that. I was also going to farm some of that out. It will be much faster for someone else to add the RHEL ecosystem information, for example.

Reply:
I'm good with that.

Reply:
Hey, @richlander I can help with writing up the dependencies for RHEL/Fedora. Is there an application/test case I can use to verify that my list of dependencies is minimal but sufficient?